site/_config.yml

```diff
@@ -121,6 +121,12 @@ defaults:
       gh: https://github.com/heptio/velero/tree/v0.3.0
       layout: "docs"
 
+page_gen:
```

Comment:
So I think I've pieced this together - this tells the pagegen plugin to use the data in the `shortlinks` file to generate a page per entry, using the `redirect.html` template.

Adding in new redirects should be as simple as adding the entry into `shortlinks.yml`, right?

Reply:
Yup, you got it!